libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -1298,6 +1296,72 @@ public function testAllowedDestinationConfigurations(
         self::assertSame($expectedTables, $tablesIds);
     }
 
+    #[NeedsEmptyOutputBucket]
+    public function testWriteAlwaysFlag(): void
+    {
+        if ($this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
+            self::markTestSkipped('Write always flag is not implemented for slice feature.');
+        }
+
+        $this->testAllowedDestinationConfigurations(
+            null,
+            null,
+            [
+                [
+                    'source' => 'table.csv',
+                    'destination' => '%s.table1',
+                    'write_always' => false,
+                ],
+                [
+                    'source' => 'table.csv',
+                    'destination' => '%s.table2',
+                    'write_always' => true,
+                ],
+            ],
+            ['%s.table2'],
+            true,
+        );
+    }
+
+    #[NeedsEmptyOutputBucket]
+    public function testAllowedMultipleMappingsOfSameSource(): void
+    {
+        if ($this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
+            self::markTestSkipped('This test is not relevant for slice feature.');
+        }
+
+        $this->testAllowedDestinationConfigurations(
+            null,
+            null,
+            [
+                ['source' => 'table.csv', 'destination' => '%s.table1'],
+                ['source' => 'table.csv', 'destination' => '%s.table2'],
+            ],
+            ['%s.table1', '%s.table2'],
+        );
+    }
+
+    #[NeedsEmptyOutputBucket]
+    public function testForbiddenMultipleMappingsOfSameSource(): void
+    {
+        if (!$this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
+            self::markTestSkipped('This test is relevant only for slice feature.');
```

Comment:
Tahle konfigurace na slice feature nejde, protoze ma oboji stejnej source.

Kdyz source zmenim aby kazdej source byl jinej, tak to nic nezapise.
Musim si ten write always nastudovat co vlastne dela a pripadne na to zalozim dalsi issue